README.md

```diff
@@ -9,63 +9,93 @@
 
-* Linux/macOS/Windows
-* [Docker](https://www.docker.com/)
-* [I-On Core](https://github.com/i-on-project/core/blob/master/README.md#running) running
-* [PostgreSQL](https://www.postgresql.org/) instance running with database **spring_batch** created. 
+### Architecture
+
+![I-On Integration Architecture](img/ion_integration_architecture.png)
+
+**I-On Integration** uses *batch processing* techniques to acquire and process all unstructured data and write it to the common **File Repository** (a Git repo whose sole purpose is to host this data).
+
+The **Scheduler** component, as the name suggests, is responsible for periodically triggering job executions through Integration's Web API (not yet available).
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;With Docker installed just use `docker run --name postgres -e POSTGRES_PASSWORD=1234 -e POSTGRES_DB=spring_batch -p 5432:5432 -d postgres`
+### Built With
+Presently I-On integration is built using the following main frameworks and technologies:
+- [Spring Batch](https://spring.io/projects/spring-batch) for **batch processing** and job maintenance
+- [Spring Boot Starter Web](https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-web) for it's **Web API** (not yet available)
+- **Github** for **version control**, **CI/CD** pipelines and to host the **File Repository** in a separate [Github Repo](https://github.com/i-on-project/integration-data)
+- [Kotlin](https://kotlinlang.org/) as the main programming language
+- [Docker](https://www.docker.com/) for **containerization** of deployed artifacts
 
-#### Application.properties
+## Getting Started
+Getting your own copy of **I-On Integration** running locally while not difficult can be tedious if done manually so we've provided a simple **Docker Compose** script to simplify it.
 
-Before running some variables have to be updated, such as `spring.datasource.url`, `spring.datasource.username` and `spring.datasource.password`
+### Requirements
 
-If the previous docker command was used, the values to use would be:
+* [Docker](https://www.docker.com/)
+* [Docker Compose](https://docs.docker.com/compose/install/) (**Note**: Already included in **Docker Desktop for Windows**)
+* [Git](https://git-scm.com/) (Optional) to clone the repository, although you can also **download it as a zip file** from the top of this page
+
+### Running with Docker Compose
+Navigate to the **directory** in your local file system where you've chosen to place the source code in and, using your favorite **terminal**, run the following command: 
+```
+docker-compose up
+```
+This mode will run all the application containers and bind **their output to your standard output**. If you want to run them in the **background** simply add the **detach** flag to the command:
 ```
-spring.datasource.url=jdbc:postgresql://127.0.0.1:5432/spring_batch
-spring.datasource.username=postgres
-spring.datasource.password=1234 
+docker-compose up -d
 ```
```

Comment:
After bringing up the containers, it is possible to interact with i-on integration. Make it explicit here what are the ports used to access the git server or any other available service.

Reply:
Added an explicit listing of available ports as suggested. They were only listed in the "Customizing" section which was, in retrospect, a mistake.